metpy/calc/thermo.py

```diff
@@ -439,17 +443,30 @@ def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None, dewpt_sta
         if not any(idx):
             x, y = this_lcl
             return x, y
-        # Otherwise, make select first candidate LFC above the LCL
+        # Otherwise, find all LFCs that exist above the LCL
+        # What is returned depends on which flag as described in the docstring
         else:
-            x = x[idx]
-            y = y[idx]
-            return x[0], y[0]
+            x, y = _multiple_el_lfc_options(x, y, idx, which)
```

Comment:
Oops, I missed this one before for combining into
```python
return _multiple_el_lfc_options(x, y, idx, which)
```

Reply:
One would think that I would remember that everything is parallel and I would catch these the first time around too. Updated and pushed.

Reply:
Well, it took me 3 reviews to get it all, so clearly I've had my "A" game lately...